### ISSUE TITLE
[Ads] gogoanime.se

### ISSUE THREAD

**Issue by jspenguin2017** [uBlockOrigin/uAssets#2256]
### URL(s) where the issue occurs

`https://www2.gogoanime.se/`

### Describe the issue

Ads are showing, randomized id and stuff, may need CSS matching to block. 

### Screenshot(s)

![image](https://user-images.githubusercontent.com/7283682/39958821-6d55dfc6-55c5-11e8-9c35-d658be6c68a8.png)
![image](https://user-images.githubusercontent.com/7283682/39958831-7ae7a462-55c5-11e8-8595-4303a4b612d8.png)

### Versions

- Browser/version: Chrome 68
- uBlock Origin version: 1.16.4

### Settings

All default

### Notes

Original report: 
```
Reference: Quick Issue Reporter

Message:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36
Nano Defender 14.18

[Ads] https://www2.gogoanime.se/
adds were successfully blocked before, but now two banners either side and an internal one are showing, manually zapping them had no effect on page reload, there back, please remedy :)
```

**Comment by mapx-** [comment]
or ?
`||www2.gogoanime.se^$script`

**Comment by ghajini** [comment]
videos not shown, all thumbnail images getting blocked
site is broken due to easylist filters/other filters 
can you see @mapx-

**Comment by mapx-** [comment]
this 1 is working for me:
`/^https?:\/\/([0-9a-z\-]+\.)?(9anime|gogoanime|kimcartoon|memecenter|readcomiconline|watchcartoononline)\.[a-z]{2,4}\/(?!(Content|Uploads|static|thumbs|player|wp-content|wp-includes))(.*)/$image,badfilter`

**Comment by mapx-** [comment]
@ryanbr

**Comment by ghajini** [comment]
that badfilter is for all sites?

**Comment by mapx-** [comment]
or use the filter above and add 
`/^https?:\/\/([0-9a-z\-]+\.)?(9anime|kimcartoon|memecenter|readcomiconline|watchcartoononline)\.[a-z]{2,4}\/(?!(Content|Uploads|static|thumbs|player|wp-content|wp-includes))(.*)/$image`

**Comment by ghajini** [comment]
alll works except site is loading very slowly after that filter

**Comment by KiLL4FuNK** [comment]
Pop-up play video

`https://www1.gogoanime.sh/kuroko-basuke-2-episode-11`

**Comment by ZaphodBeebblebrox** [comment]
Fixed in e97cbdd. It's caused by the video palyer, vidstream.io.
